packages/gateway/src/config.ts

```diff
@@ -146,8 +146,12 @@ export async function getBuiltinPluginsFromConfig(
 
 export async function getCacheInstanceFromConfig(
   config: GatewayCLIBuiltinPluginConfig,
-  ctx: Pick<GatewayConfigContext, 'logger' | 'pubsub'>,
+  ctx: GatewayConfigContext,
```

Comment:
@ardatan @enisdenjo any reason to limit the signature here? I wasn't sure if it's needed to change or not.

Reply:
In order to pass the things needed, since we have a factory function to be passed, it doesn't make much sense. But maybe we can still exclude `cache`.